iib/workers/config.py

```diff
@@ -46,6 +46,8 @@ class Config(object):
         'iib.workers.tasks.build_create_empty_index',
         'iib.workers.tasks.general',
     ]
+    # Path to hidden location of SQLite database
+    hidden_index_db_path = '/_hidden/do.not.edit.db'
```

Comment:
@gallettilance I modified the path for index.db based discussion on yesterdays meeting. To be explicit that this should not be edited manually. Is this sufficient? Or would you suggest different name/location... ?
Thank you!

Reply:
Should this probably reside under /var/lib/iib/_hidden/do.not.edit.db?

Reply:
I agree, this is better path. Thanks Yash!